rnd/autogpt_server/autogpt_server/executor/manager.py

```diff
@@ -42,6 +45,8 @@ def execute_node(loop: asyncio.AbstractEventLoop, data: Execution) -> ExecutionS
     Returns:
         The subsequent node to be enqueued, or None if there is no subsequent node.
     """
+    from autogpt_server.server.server import AgentServer
+    agent_server_client = get_service_client(AgentServer) 
```

Comment:
I made this change when I was having an issue with the tests. I'm going to try the property type again to see if was just an issue with the test.